README.rst

```diff
@@ -8,20 +8,47 @@ inline-plz
 .. image:: https://img.shields.io/travis/guykisel/inline-plz.svg
         :target: https://travis-ci.org/guykisel/inline-plz
 
-.. image:: https://readthedocs.org/projects/inline-plz/badge/?version=latest
-        :target: https://readthedocs.org/projects/inline-plz/?badge=latest
-        :alt: Documentation Status
 
-
-Inline your lint messages
+Tired of reading through CI console logs to find your lint errors? Inline your lint messages in your diffs!
 
 * Free software: ISC license
-* Documentation: https://inline-plz.readthedocs.org.
 
 Features
 --------
 
-* TODO
+* Run linters against your code and comment in your diffs at the failing lines
+* Automatically run linters with reasonable default configs
+* Easy to add new linter configurations
+
+How to use
+--------
+
+```
```

Comment:
`rst-lint: Possible incomplete section title.
Treating the overline as ordinary text because it's so short.`